src/common/AmazonS3Backend.ts

```diff
@@ -293,4 +293,18 @@ export class AmazonS3Backend implements IBackend {
     });
   }
 
+  /**
+   * Initiate S3 multi-part upload using ManagedUpload
+   */
+  public initMultipartUpload(params: any): Promise<any> {
```

Comment:
Using the params dictionary as an argument makes it unclear to me how this works.  Would it make sense to use Bucket, Key, and Body as keyword arguments?  We should also consider passing the folder and file to match the style of updateFormData.

Reply:
Thanks for pointing this out. AmazonS3Backend implements the IBackend interface, which is supposed to be generic across all backend types. To keep this function generic, it accepts a single params object (since bucket, key, body would be very specific to S3).

I'll update the 'initMultipartUpload' description in AmazonS3Backend to make clear what the expect params object should look like.